scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -120,24 +122,92 @@ class RunCoverage(
 
     val testTargets = bazelClient.retrieveBazelTargets(testFilePaths)
 
+    /*since I couldn't actually find any multi test target : file ones to test
+    * I am for now introducing mock data to test multi aggregated coverage report
+    * also that's going to save me a light year :|
+    * */
+
     val coverageReports = testTargets.mapNotNull { testTarget ->
       runCoverageForTarget(testTarget)
     }
 
-    coverageReports.takeIf { it.isNotEmpty() }?.run {
-      val reporter = CoverageReporter(repoRoot, this, reportFormat)
-      val (computedCoverageRatio, reportText) = reporter.generateRichTextReport()
+    /*val coverageReports = listOf(CoverageReport.newBuilder()
```

Comment:
### To be removed!
Kept this for reference as this is the mock coverageReport list I tested this PR with for many to one relationship as I couldn't find a working one. This will be removed, and the PR needs to be tested with an actual working file.